sample_information_validator.py

```diff
@@ -23,78 +23,280 @@
                     fmt='%(asctime)s %(levelname)s %(message)s')
 
 
-class NumericValidator(object):
-    """NumericValidator
+class ProjectSheet:
+#Class Attributes
+    SHEET_NAME = 'Sample information'
+    FIRST_LINE = 20  # First line where user submitted data is located
+    SAMPLE_NAME_COL = 'O'
+    A_RATIO_COL = 'R'  # A260:A280 ratio
+    CONC_COL = 'S'
+    VOL_COL = 'T'
+    RIN_COL = 'V'
+    SAMPLE_TYPE = 'O8'
+    PLATE_ID = 'M6'
+    PROJECT_NAME_USER_SHEET = 'M3'
 
-    Checks whether cell is numeric or not.
-    Takes optional params:
-       decimal_comma - Checks whether swedish comma is used or not.
-                       [default False]
-    """
+    # Initializer / Instance attributes
+    def __init__(self, sample_info_sheet):
+        self.sample_info_sheet = sample_info_sheet
+        self.work_sheet = None
+        self.sample_rec = None
 
-    def validate(self, cell):
-        """Checks whether value is numeric or not."""
-        logging.debug(cell.value, cell.data_type)
-        if cell.data_type != 'n':
+    # instance methods
+    def getAccessUserSheet(self):
+        if self.work_sheet == None:
```

Comment:
while this is 100% correct. Using `self.work_sheet is None` will make you look like a Pythonista.